.gitignore

```diff
@@ -7,3 +7,6 @@
 /.DS_Store
 .env
 .ruby-version
+
+# It's good practice to not include Gemfile.lock in vc for gems; catch dependency issues earlier
+Gemfile.lock
```

Comment:
Hm, I think we _do_ want to commit the Gemfile.lock.

e.g. Rails does that:
https://github.com/rails/rails/blob/v7.1.3.4/Gemfile.lock

Also comment from a while back for devise:
https://github.com/heartcombo/devise/pull/3147#issuecomment-52193788

And [current Bundler docs say to check it in](https://bundler.io/guides/faq.html#using-gemfiles-inside-gems):

> Q: Should I commit my Gemfile.lock when writing a gem?
>
> A: Yes, you should commit it. The presence of a Gemfile.lock in a gem’s repository ensures that a fresh checkout of the repository uses the exact same set of dependencies every time. We believe this makes repositories more friendly towards new and existing contributors. Ideally, anyone should be able to clone the repo, run bundle install, and have passing tests. If you don’t check in your Gemfile.lock, new contributors can get different versions of your dependencies, and run into failing tests that they don’t know how to fix.
>
> Q: But I have read that gems should not check in the Gemfile.lock!
>
> A: The main advantage of not checking in your Gemfile.lock is that new checkouts (including CI) will immediately have failing tests if one of your dependencies changes in a breaking way. Instead of forcing every fresh checkout (and possible new contributor) to encounter broken builds, the Bundler team recommends either using a tool like Dependabot to automatically create a PR and run the test suite any time your dependencies release new versions. If you don’t want to use a dependency monitoring bot, we suggest creating an additional daily CI build that deletes the Gemfile.lock before running bundle install. That way you, and others monitoring your CI status, will be the first to know about any failures from dependency changes.

Reply:
So, maybe we should keep it - but run dependabot on this repo? There aren't a ton of dependencies

Reply:
👍 adding dependabot seems like a good idea to me.